core/templates/components/skill-mastery/skill-mastery.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="oppia-skill-mastery-viewer" *ngIf="!loadingMessage">
+  <div class="skill-mastery-degree">
+    <b> Mastery Degree: </b>
+    <div style="margin-bottom: 10px">
```

Comment:
Avoid inline styling here and below in this file.

Reply:
@Hudda this should have been caught by the linter right?

Reply:
The PR for this check is WIP, will be implemented soon.